document_page_approval/data/email_template.xml

```diff
@@ -1,20 +1,17 @@
 <?xml version="1.0"?>
-<odoo>
+<odoo noupdate="1">
 
     <!-- Allow user to make upgrade-proof customizations to email template -->
-
-    <data noupdate="1">
-
-        <record id="email_template_new_draft_need_approval" model="mail.template">
-            <field name="name">Automated new draft need approval Notification Mail</field>
-            <field name="email_from">${object.create_uid.company_id.email or 'noreply@localhost.com'}</field>
-            <field name="subject">New version of "${object.page_id.name}" to approve</field>
-            <field name="email_to">${object.get_approvers_email}</field>
-            <field name="model_id" ref="model_document_page_history"/>
-            <field name="auto_delete" eval="True"/>
-            <field name="lang">${object.create_uid.partner_id.lang}</field>
-            <field name="body_html">
-                <![CDATA[
+    <record id="email_template_new_draft_need_approval" model="mail.template">
+        <field name="name">Automated new draft need approval Notification Mail</field>
+        <field name="email_from">${object.create_uid.company_id.email or 'noreply@localhost.com'}</field>
+        <field name="subject">New version of "${object.page_id.name}" to approve</field>
+        <field name="email_to">${object.get_approvers_email}</field>
+        <field name="model_id" ref="model_document_page_history"/>
+        <field name="auto_delete" eval="True"/>
+        <field name="lang">${object.create_uid.partner_id.lang}</field>
```

Comment:
@LeartS There could be many approvers with different languages. If they need to approve the content, I assumed they can at least read in the language of the creator.